docs/integrations/python.md

```diff
@@ -31,119 +34,89 @@ Install the Python client using pip:
 
 
 ```shell
-pip install 'lakefs_sdk~=1.0'
+pip install lakefs
 ```
 
 ### Initializing
 
+The High Level SDK by default will try to collect authentication parameters from the environment and attempt to create a default client.
+When working in an environment where **lakectl** is configured it is not necessary to instantiate a lakeFS client or provide it for creating the lakeFS objects.
+In case no authentication parameters exist, it is also possible to explicitly create a lakeFS client
+
 Here's how to instantiate a client:
 
 ```python
-import lakefs_sdk
-from lakefs_sdk.client import LakeFSClient
+from lakefs.client import Client
 
-configuration = lakefs_sdk.Configuration(
+clt = Client(
     host="http://localhost:8000",
     username="AKIAIOSFODNN7EXAMPLE",
     password="wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
 )
-client = LakeFSClient(configuration)
-``` 
-
-For testing SSL endpoints you may wish to use a self-signed certificate.  If you do this and receive an `SSL: CERTIFICATE_VERIFY_FAILED` error message you might add the following configuration to your client:
-
-```python
-configuration.verify_ssl = False
 ```
 
-{: .warning }
-This setting allows well-known "man-in-the-middle",
-impersonation, and credential stealing attacks.  Never use this in any
-production setting.
-
-Optionally, to enable communication via proxies, simply set the proxy configuration:
-
-```python
-configuration.ssl_ca_cert = <path to a file of concatenated CA certificates in PEM format> # Set this to customize the certificate file to verify the peer
-configuration.proxy = <proxy server URL>
-``` 
```

Comment:
Don't think we need to remove this part - it was added after iddo had to resolve issues with users that had issues working with custom certificates and web proxy.